x/gov/client/rest/util.go

```diff
@@ -100,3 +102,15 @@ func signAndBuild(w http.ResponseWriter, cliCtx context.CLIContext, baseReq base
 
 	w.Write(output)
 }
+
+func parseInt64OrReturnBadRequest(s string, w http.ResponseWriter) (n int64, ok bool) {
```

Comment:
Should this be accessible from `cosmos-sdk/client` instead ?

Reply:
Most of this `util.go` file should move to there, but I'll do that in a separate PR.